Examples/IcmpFileTransfer/IcmpFileTransfer-pitcher.cpp

```diff
@@ -36,12 +36,13 @@ void usleep(__int64 usec)
 	LARGE_INTEGER ft;
 
 	ft.QuadPart = -(10 * usec);  // Convert to 100 nanosecond interval, negative value indicates relative time
-
+    //NULL is Used for WindowAPIs and no need to change to nullptr for C++11
```

Comment:
Indentation not correct for added all comment lines

Reply:
Yeah, I fixed them , should I raise another PR?

Reply:
No need for new PR. All PRs are squash merged so you can add any commits to this PR if related.